packages/ember-glimmer/tests/integration/components/local-lookup-test.js

```diff
@@ -48,6 +48,20 @@ moduleFor('Components test: local lookup', class extends RenderingTest {
     this.assertText('Nested template says: Hi!', 'Re-render works');
   }
 
+  ['@test moduleName remains unchanged']() {
+    this.registerComponent('x-outer/x-inner', { template: 'Nested template says: {{yield}}' });
+    this.registerComponent('x-outer', { template: '{{#x-inner}}Hi!{{/x-inner}}' });
+    this.registerComponent('x-root', { template: '{{x-outer}}' });
+
+    this.render('{{x-root}}');
+
+    let moduleName = this.owner.lookup('template:-top-level').meta.moduleName;
```

Comment:
Looks like `this.owner.lookup('template:-top-level')` is returning `undefined` (causing the Travis failures).